sdk/servicefabric/arm-servicefabric-rest/package.json

```diff
@@ -13,21 +13,18 @@
     "isomorphic"
   ],
   "license": "MIT",
-  "main": "dist/index.js",
-  "module": "./dist-esm/src/index.js",
-  "types": "./types/arm-servicefabric.d.ts",
+  "main": "./dist/commonjs/index.js",
```

Comment:
Since this is a move to core-lro v3, shouldn't this be a major version bump since the two are incompatible?

Reply:
This library is still in preview, I think it's fine to just bump it to beta.2 ?